debian/ubuntu-advantage-tools.postinst

```diff
@@ -382,6 +382,23 @@ case "$1" in
       if dpkg --compare-versions "$PREVIOUS_PKG_VER" lt "27.13~"; then
           cleanup_apt_news_flag_file
       fi
+
+      
+      # LP: #2003977
+      # The version gate is open ended, and should be closed when the apt_news
+      # configuration is moved away from a conffile.
+      if dpkg --compare-versions "$2" ge 27.11.3; then
```

Comment:
I think this needs to be compared to `27.11.3~`, since all u-a-t 27.11.3 releases are of the form `27.11.3~YY.MM.N`. Let's see if others agree.

Reply:
@panlinux fixed